### ISSUE TITLE
mvdXML v1.2 schema problems no.2?

### ISSUE THREAD

**Issue by pjanck** [buildingSMART/mvdXML#9]
I've taken the current Draft9 of the schema (https://github.com/buildingSMART/mvdXML/blob/master/mvdXML1.2/xsd/mvdXML_V1-2_Draft9.xsd), with the fixes in #8, and ran it through VisualStudio xsd utility. I get this error (sorry for German language):

```
Schema validation warning: Verweis auf eine nicht deklarierte Modellgruppe 'https://standards.buildingsmart.org/IFC/RELEASE/IFC4/Add2TC1:IfcUnit'. Line 161, position 11.
```

It seems that this part makes problems:

https://github.com/buildingSMART/mvdXML/blob/6677c83f52f1d7cec126e70ae5cb0bd7c27c9aa3/mvdXML1.2/xsd/mvdXML_V1-2_Draft9.xsd#L148-L172

**Comment by pjanck** [comment]
A wild guess would be, that the `ifc` namespace cannot be resolved; however, I'm unsure as to how to fix this.

**Comment by MatthiasWeise** [comment]
We are using the same namespace as in ifcXML.xsd of IFC4 Add2 TC1 (https://standards.buildingsmart.org/IFC/RELEASE/IFC4/ADD2_TC1/XML/IFC4.xsd), but did not provide a schema location for download. Can you try to add the following statement to the <xs:schema ..> tag?

`ifc:schemaLocation="https://standards.buildingsmart.org/IFC/RELEASE/IFC4/Add2TC1 https://standards.buildingsmart.org/IFC/RELEASE/IFC4/ADD2_TC1/XML/IFC4.xsd"`

Full statement then: 
`<xs:schema xmlns:mvd="https://standards.buildingsmart.org/MVD/RELEASE/mvdXML/v1-2/" xmlns:xs="http://www.w3.org/2001/XMLSchema" targetNamespace="https://standards.buildingsmart.org/MVD/RELEASE/mvdXML/v1-2/" elementFormDefault="qualified" attributeFormDefault="unqualified" xmlns:ifc="https://standards.buildingsmart.org/IFC/RELEASE/IFC4/Add2TC1" version="1.2">`

**Comment by pjanck** [comment]
I don't see the added tag in the full statement - care to double-check?

**Comment by MatthiasWeise** [comment]
Sorry, it seems I copied the old one

`<xs:schema xmlns:mvd="https://standards.buildingsmart.org/MVD/RELEASE/mvdXML/v1-2/" xmlns:xs="http://www.w3.org/2001/XMLSchema" targetNamespace="https://standards.buildingsmart.org/MVD/RELEASE/mvdXML/v1-2/" elementFormDefault="qualified" attributeFormDefault="unqualified" xmlns:ifc="https://standards.buildingsmart.org/IFC/RELEASE/IFC4/Add2TC1" ifc:schemaLocation="https://standards.buildingsmart.org/IFC/RELEASE/IFC4/Add2TC1 https://standards.buildingsmart.org/IFC/RELEASE/IFC4/ADD2_TC1/XML/IFC4.xsd" version="1.2">`

**Comment by pjanck** [comment]
Nope, still receiving the same error. Do you wish to make a live exchange? I can call you.

**Comment by SergejMuhic** [comment]
Did a little digging. Let us try this tomorrow:

`<xs:import namespace=` before defining `xs:schema`.
